pynanoleaf/pynanoleaf.py

```diff
@@ -32,6 +34,16 @@ def _dictitem_property(keystr):
         fset=_deepsetter(key)
     )
 
+def _parseEffect(effect):
+    """These effect names have special meaning and are not directly setable
+    and should not be returned as effect name for Consumers."""
+    RESERVED_EFFECT_NAMES = ["*Solid*", "*Static*", "*Dynamic*"]
+    try:
```

Comment:
Any particular reason a try/except is used here? If not, could you please change it to something like the following:
```python
if effect in RESERVED_EFFECT_NAMES
  return None
else
  return effect
```

Reply:
@Taosmi this is still open, probably an oversight. Could you please double-check?

Reply:
ok